### ISSUE TITLE
Training from scratch (200k iters + 100k iters) doesn't achieve the reported performance (PSNR 35.46) with GoPro

### ISSUE THREAD

**Issue by ohjinjin** [AHupuJR/EFNet#18]
Hello, thank you for your good research first of all.

I was trying to reproduce the performance reported in your paper with SCER-GoPro dataset that you shared as a link.
(Before I started training from scratch, I had checked that your pre-trained weights gave me PSNR 35.44.
I thought this difference was not that big.)

Since I trained by myself with SCER-GoPro dataset and the code implementation here, total number of iterations was set as 200k which was not equal to the explanation in your paper. (The paper said total num of iter was 300k). Hence I thought this inconsistency was from that difference.
Thus I proceeded to train an additional 100k iters again, however, the performance became lower than that of 200k iter.

Is it because of issues from loading the resume training or should I have to modify some part of the experimental setting in code implementation to obtain the same performance?
It would be really appreciated if you answer my question!

**Comment by ohjinjin** [comment]
I figured out that i made a mistake about setting batchsize and the number of  gpus. Thanks!

**Comment by shkids** [comment]
hi, ohjinjin
How did you set up yml? If you're trying to train, you'll need to run the test after 258 iterations, depending on your setup. However, unlike the settings, there is a problem where the test starts as soon as iter rotates 8 times.

**Comment by tgytgytgy123456-ctrl** [comment]
> I figured out that i made a mistake about setting batchsize and the number of gpus. Thanks!

Hello! Thank you very much for your efforts in reproducing the results of this excellent work.I have also been trying to reproduce the model using the same SCER‑GoPro dataset and code, but my results at 200k iterations are:PSNR: 35.0503, SSIM: 0.9694,which are quite different from those reported in the paper.
I suspect there might be an issue with my configuration file, but I have carefully checked all the key training parameters mentioned in the paper (such as patch size, learning rate, batch size, etc.) and found no obvious discrepancies.
Could you please share the detailed settings of your configuration file so that I can compare and figure out why there’s a discrepancy in my results
